server/base/src/main/java/org/apache/accumulo/server/zookeeper/TransactionWatcher.java

```diff
@@ -122,28 +122,37 @@ public boolean transactionComplete(String type, long tid) throws Exception {
     }
   }
 
+  public void runQuietly(String ztxBulk, long tid, Runnable task) {
```

Comment:
Can you add a javadoc description of what this run method does?

Reply:
Added in f2d62d7